security/pom.xml

```diff
@@ -49,6 +49,10 @@
             <artifactId>jmock-legacy</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.kaazing</groupId>
```

Comment:
It is present in the parent gateway pom.xml in the dependencyManagement section with scope test.  I can add test scope for readability.